source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Controllers/EvaluationsController.cs

```diff
@@ -49,6 +49,8 @@ private string DetectUser()
         [CommonJobsAuthorize(Roles = "EmployeeManagers")]
         public ActionResult PeriodCreation(string period){
             ViewBag.Period = period;
+            ViewBag.IsPeriodCreation = true;
+            ViewBag.Active = "Create";
```

Comment:
What if we call this "ActiveSection" or something that specifies what is it that is active?